### PR TITLE
[fix] align loss scale to DiT official impl

### LINKED ISSUE #16: Maybe need accelerator.reduce? Loss scale mismatch DiT official code
DiT official code loss log
image

while Fast DiT loss is much smaller than official loss 
image

I think maybe fast dit code miss gathering the loss across all GPUs. After using accelerator.reduce
`avg_loss = accelerator.reduce(avg_loss, reduction="sum")`
the loss matches the result of official code 
image

### PR DESCRIPTION
**Pull request by henry-y**
## Summary

This PR aligns the print loss behavior with the official DiT implementation.

## Details

- Uses accelerate.reduce to properly aggregate the loss across all processes when using distributed training, ensuring that the reported loss matches the official DiT implementation.

## Related Issue

Resolves [#16](https://github.com/chuanyangjin/fast-DiT/issues/16)

### CODE REVIEW

**Comment by henry-y** [comment]
@chuanyangjin

**Comment by chuanyangjin** [comment]
Looks good. Thanks!
